spec/controllers/works_wizard_controller_spec.rb

```diff
@@ -101,7 +101,6 @@
       it "redirects to file-upload" do
         post(:readme_uploaded, params:)
         expect(response.status).to be 302
-        expect(fake_readme).to have_received(:attach)
```

Comment:
No file is uploaded in `readme_uploaded`, files are now uploaded in `readme_uploaded_payload`. See new tests below.